src/backends/interflop-vprec/interflop_vprec_vector.h

```diff
@@ -0,0 +1,588 @@
+#ifndef _INTERFLOP_VPREC_VECTOR_H_
+#define _INTERFLOP_VPREC_VECTOR_H_
+
+/******************** VPREC HELPER FUNCTIONS *******************
+ * The following functions are used to set virtual precision,
+ * VPREC mode of operation and instrumentation mode.
+ ***************************************************************/
+
+/* Avoid warning about vector argument of type X without 'set' enabled changes
+   the ABI */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpsabi"
+
```

Comment:
I have a warning at each functions which use vectors types beyond the max vector width supported.
Then I decided to put a diagnostic in the entire file to fix it.